doc/classes/AABB.xml

```diff
@@ -33,211 +33,314 @@
 			<description>
-				Returns [code]true[/code] if the [AABB] contains a point. Points on the faces of the AABB are considered included, though float-point precision errors may impact the accuracy of such checks.
-				[b]Note:[/b] This method is not reliable for [AABB] with a [i]negative size[/i]. Use [method abs] to get a positive sized equivalent [AABB] to check for contained points.
+				Returns [code]true[/code] if the bounding box contains the given [param point]. By convention, points exactly on the right, top, and front sides are [b]not[/b] included.
+				[b]Note:[/b] This method is not reliable for [AABB] with a [i]negative[/i] [member size]. Use [method abs] first to get a valid bounding box.
 			</description>
 		</method>
 		<method name="has_surface" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the [AABB] has a surface or a length, and [code]false[/code] if the [AABB] is empty (all components of [member size] are zero or negative).
+				Returns [code]true[/code] if this bounding box has a surface or a length, that is, at least one component of [member size] is greater than [code]0[/code]. Otherwise, returns [code]false[/code].
 			</description>
 		</method>
 		<method name="has_volume" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the [AABB] has a volume, and [code]false[/code] if the [AABB] is flat, empty, or has a negative [member size].
+				Returns [code]true[/code] if this bounding box's width, height, and depth are all positive. See also [method get_volume].
 			</description>
 		</method>
 		<method name="intersection" qualifiers="const">
 			<return type="AABB" />
 			<param index="0" name="with" type="AABB" />
 			<description>
-				Returns the intersection between two [AABB]. An empty AABB (size [code](0, 0, 0)[/code]) is returned on failure.
+				Returns the intersection between this bounding box and [param with]. If the boxes do not intersect, returns an empty [AABB]. If the boxes intersect at the edge, returns a flat [AABB] with no volume (see [method has_surface] and [method has_volume]).
+				[codeblocks]
+				[gdscript]
+				var box1 = AABB(Vector3(0, 0, 0), Vector3(5, 2, 8))
+				var box2 = AABB(Vector3(2, 0, 2), Vector3(8, 4, 4))
+
+				var intersection = box1.intersection(box2)
+				print(intersection.position) # Prints (2, 0, 2)
+				print(intersection.size)     # Prints (3, 2, 4)
+				[/gdscript]
+				[csharp]
+				var box1 = new AABB(new Vector3(0, 0, 0), new Vector3(5, 2, 8));
+				var box2 = new AABB(new Vector3(2, 0, 2), new Vector3(8, 4, 4));
+
+				var intersection = box1.intersection(box2);
+				GD.Print(intersection.position); // Prints (2, 0, 2)
+				GD.Print(intersection.size);     // Prints (3, 2, 4)
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] If you only need to know whether two bounding boxes are overlapping, use [method intersects], instead.
```

Comment:
```suggestion
				[b]Note:[/b] If you only need to know whether two bounding boxes are intersecting, use [method intersects], instead.
```
I'd use the same term consistently here, while we don't have a separate "overlaps" method the two terms aren't necessarily equivalent in geometry code elsewhere.